### DISCUSSION TITLE
Code in .mdx files doesn't highlighted correctly

### DISCUSSION THREAD

**Discussion by elizavetaKrainova** [mdx-js/mdx#2197] [Q&A] [answered]
Hello! 

I have an issue with .mdx files. I use React 16 and Storybook. And the highlighting stops working right after I put the word "for" in the text. If I remove this word then it works fine. Also I can't put the comment which starts with "<!--". It brakes everything. 

This issue happens only on my machine. My colleagues don't have it. I thought that maybe there is some special extensions or settings that I also should use to make it work correctly. 

Unfortunately, I can't attach any screenshots because it's not my personal project. The most mysterious thing is that it works for my colleague but not for me. 

Looking forward for your reply, thank you!

**Comment by ChristianMurphy** [comment, marked as answer]
Welcome @elizavetaKrainova! 👋 
Sorry you ran into a spot of trouble.
It's a bit difficult to offer much specific on this without any detail.

An initial question, where are you looking for highlighting?
Inside storybook itself (and if so which panel inside storybook)? Inside an editor like VSCode or Intellij? Somewhere else?

> This issue happens only on my machine. My colleagues don't have it. I thought that maybe there is some special extensions or settings that I also should use to make it work correctly.

From this I'm guessing you are inside VSCode?
If so check to see if you are running the official VSCode extension https://marketplace.visualstudio.com/items?itemName=unifiedjs.vscode-mdx

If the issue persists with the official plugin, consider raising an issue https://github.com/mdx-js/vscode-mdx/issues and sharing a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) snippet of MDX text which shows the issue (removing any confidential text, using something generic like [lorem ipsum](https://en.wikipedia.org/wiki/Lorem_ipsum) instead).
